iree/turbine/kernel/wave/templates/quantized_attention.py

```diff
@@ -0,0 +1,225 @@
+# Copyright 2025 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import torch
+import iree.turbine.kernel.lang as tkl
+import iree.turbine.kernel.wave as tkw
+from iree.turbine.kernel.lang.global_symbols import *
+from iree.turbine.kernel.wave.constraints import MMAType
+from iree.turbine.kernel.wave.utils.mma_utils import (
+    get_mfma_load_elems_per_thread,
+    get_mfma_store_elems_per_thread,
+)
+from .attention_common import AttentionShape
+import math
+from iree.turbine.kernel.wave.utils.general_utils import torch_dtype_to_wave
+
+
+def get_brevitas_pertensor_fp8_attention_kernel(
+    shape: AttentionShape,
+    mfma_variant: MMAType,
+    logit_dtype: torch.dtype = torch.float16,
+    f8_dtpye: torch.dtype = torch.float8_e4m3fnuz,
+    dynamic_dims: bool = False,
+    is_causal: bool = False,
+    q_scale=1.0,
+    k_scale=1.0,
+    v_scale=1.0,
+):
+    # Input sizes
+    B = tkl.sym.B
```

Comment:
Can you switch these symbols to N_Q, N_KV, etc. so we have a standard notation for attention templates?

Reply:
sg!

Reply:
Done!